mod-pubsub-server/pom.xml

```diff
@@ -175,7 +175,7 @@
             <annotationProcessor>lombok.launch.AnnotationProcessorHider$AnnotationProcessor</annotationProcessor>
             <annotationProcessor>io.vertx.codegen.CodeGenProcessor</annotationProcessor>
           </annotationProcessors>
-          <generatedSourcesDirectory>${project.build.directory}/generated-sources/services/audit</generatedSourcesDirectory>
+          <generatedSourcesDirectory>${project.build.directory}/generated-sources/services/org/folio/services/audit</generatedSourcesDirectory>
```

Comment:
Why does this path have to change?
Just FYI, the actual path after code generation is "generated-sources/services/**audit**/org/folio/services/**audit**"

Reply:
It was a problem to built it without these changes. If it is not needed we can revert changes